scripts/gemini_document_extractor.py

```
@@ -0,0 +1,235 @@
+    print("=" * 70)
+    print(f"Document type: {result['document_type']}")
+    print(f"Document title: {result['document_title']}")
+    print(f"Total sections: {result['num_sections']}")
+    print(f"Hierarchy depth: {result['hierarchy_depth']}")
+
+    # Count by element type
+    by_type = {}
+    for sec in result["sections"]:
+        t = sec.get("element_type", "unknown")
+        by_type[t] = by_type.get(t, 0) + 1
+    print(f"\nBy element type:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)